### PR TITLE
feat: add support Cypress on arm64 architecture and update snapshots

### PR DESCRIPTION
**Pull request by neretin-trike**
:baby_chick: Download canary assets:

[colors--canary.204.3408411288.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.204.3408411288.xml)  
[color_sbermarket_ios-swift--canary.204.3408411288.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.204.3408411288.swift)  
[color_sbermarket_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.204.3408411288.kt)  
[color_sbermarket_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.204.3408411288.ts)  
[color_sberprime_ios-swift--canary.204.3408411288.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.204.3408411288.swift)  
[color_sberprime_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.204.3408411288.kt)  
[color_sberprime_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.204.3408411288.ts)  
[PlasmaTokensColor--canary.204.3408411288.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.204.3408411288.swift)  
[typo_mage_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.204.3408411288.kt)  
[typo_mage_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.204.3408411288.ts)  
[typo_plasma_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.204.3408411288.kt)  
[typo_plasma_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.204.3408411288.ts)  
[typo_ruler_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.204.3408411288.kt)  
[typo_ruler_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.204.3408411288.ts)  
[typo_sage_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.204.3408411288.kt)  
[typo_sage_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.204.3408411288.ts)  
[typo_sbermarket_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.204.3408411288.kt)  
[typo_sbermarket_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.204.3408411288.ts)  
[typo_soulmate_kotlin--canary.204.3408411288.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.204.3408411288.kt)  
[typo_soulmate_react-native--canary.204.3408411288.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.204.3408411288.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.108.0-canary.204.3358877589.0
  npm install @salutejs/plasma-core@1.80.0-canary.204.3358877589.0
  npm install @salutejs/plasma-icons@1.105.0-canary.204.3358877589.0
  npm install @salutejs/plasma-temple@1.107.0-canary.204.3358877589.0
  npm install @salutejs/plasma-typo@0.27.0-canary.204.3358877589.0
  npm install @salutejs/plasma-ui@1.141.0-canary.204.3358877589.0
  npm install @salutejs/plasma-web@1.139.0-canary.204.3358877589.0
  npm install @salutejs/plasma-cy-utils@0.33.0-canary.204.3358877589.0
  npm install @salutejs/plasma-sb-utils@0.78.0-canary.204.3358877589.0
  # or 
  yarn add @salutejs/plasma-b2c@1.108.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-core@1.80.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-icons@1.105.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-temple@1.107.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-typo@0.27.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-ui@1.141.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-web@1.139.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-cy-utils@0.33.0-canary.204.3358877589.0
  yarn add @salutejs/plasma-sb-utils@0.78.0-canary.204.3358877589.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

**Review by Yeti-or — COMMENTED** [review]
А про что этот пр?

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-204/
  ui storybook: http://plasma.sberdevices.ru/pr-204/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-204/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-204/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-204/temple-storybook/

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 
 addMatchImageSnapshotCommand({
     customSnapshotsDir: Cypress.env('snapshotsDir'),
+    failureThreshold: 0.155,
```
Comment (Yeti-or) [inline cypress/support/commands.ts:5]: только для Cypress.env('plasma-temple')

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-204/

**Pull request closed without merging**